numpy/f2py/capi_maps.py

```diff
@@ -186,8 +186,8 @@ def load_f2cmap_file(f2cmap_file):
     if f2cmap_file is None:
         # Default value
         f2cmap_file = '.f2py_f2cmap'
-        if not os.path.isfile(f2cmap_file):
-            return
+    if not os.path.isfile(f2cmap_file):
```

Comment:
Would prefer `pathlib` to `os` where changes are being made

Reply:
Agreed, making changes